src/lib/certs.ts

```diff
@@ -6,8 +6,38 @@ import {
 	TOKEN_AUTH_JWT_ALGO,
 } from './config';
 import { b64decode } from './utils';
+import * as jsonwebtoken from 'jsonwebtoken';
 
-export const registryAuth = {
+const allowedAlgorithms = [
+	'HS256',
+	'HS384',
+	'HS512',
+	'RS256',
+	'RS384',
+	'RS512',
+	'ES256',
+	'ES384',
+	'ES512',
+	'PS256',
+	'PS384',
+	'PS512',
+	'none',
```

Comment:
We don't want to support the `none` algorithm, because this would mean one could provide a JWT which is not signed and we still accept it as valid. The other ones should be fine. The number is the size of the hash function 256-512 and the prefix defines the actual signature algorithm.

Reply:
Removed 'none'